aspoet/src/main/kotlin/com/google/androidstudiopoet/generators/bazel/BazelLang.kt

```diff
@@ -0,0 +1,116 @@
+/*
+Copyright 2018 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+ */
+
+package com.google.androidstudiopoet.generators.bazel
+
+/**
+ * Bazel uses the Starlark configuration language for BUILD files and .bzl extensions.
+ *
+ * Starlark is previously known as Skylark.
+ *
+ * https://docs.bazel.build/versions/master/skylark/language.html
+ */
+
+private const val BASE_INDENT = "    "
+
+interface Statement
```

Comment:
Can you please make `Statement` a sealed class? Same goes for `Attribute`?

Reply:
done.